lib/watcher/cpu-profile.js

```diff
@@ -28,14 +28,14 @@ exports.worker = function(handle) {
       return;
 
     try {
-      var profile = agent().metrics.stopCpuProfiling();
-
-      handle.emit({
-        cmd: 'cpu:profile-data',
-        stalls: stalls,
-        stallout: stallout,
-        profile: profile,
-        // stopTime: Date.now(), XXX could be useful, but doesn't have a user
+      agent().metrics.stopCpuProfiling(function(profile) {
```

Comment:
http://fredkschott.com/post/2014/03/understanding-error-first-callbacks-in-node-js/